### ISSUE TITLE
Patch behaviour within OSDM

### ISSUE THREAD

**Issue by timvandenberg-sqills** [UnionInternationalCheminsdeFer/OSDM#450]
To clarify, we can not find it anywhere documented, but the PATCH behavior within OSDM is expected to be Json Merge Patch.

This means:
- absent attributes are ignored
- null attributes will be removed
- present attributes will be updated
- new attributes will be added

Do we all agree this is the preferred behavior?

**Comment by ralfbayer-db** [comment]
I would agree to this.

**Comment by jspetrak** [comment]
Yes. That is how Patch Update should work regardles of what specification uses it. 👍

**Comment by CGantert345** [comment]
TODO --> Tim

**Comment by ralfbayer-db** [comment]
One of our suppliers seems to do this wrong. So I took it upon myself to update the documentation.
